cmd/tether/attach_linux.go

```diff
@@ -90,41 +90,17 @@ func rawConnectionFromSerial() (net.Conn, error) {
 	// 0 is the uninitialized value for Fd
 	if f.Fd() != 0 && terminal.IsTerminal(int(f.Fd())) {
 		log.Info("setting terminal to raw mode")
-		s, err := terminal.MakeRaw(int(f.Fd()))
+		_, err := terminal.MakeRaw(int(f.Fd()))
 		if err != nil {
 			return nil, err
 		}
-
-		log.Debugf("s = %#v", s)
 	}
 	if err := setTerminalSpeed(f.Fd()); err != nil {
 		log.Errorf("Setting terminal speed failed with %s", err)
 	}
 
-	var conn net.Conn
-
 	log.Infof("creating raw connection from ttyS0 (fd=%d)", f.Fd())
-
-	conn, err = serial.NewFileConn(f)
-	return conn, err
-}
-
-func (t *attachServerSSH) Start() error {
-	defer trace.End(trace.Begin(""))
-
-	t.m.Lock()
-	defer t.m.Unlock()
-
-	var err error
-
-	rand.Reader, err = os.Open(pathPrefix + "/urandom")
```

Comment:
https://golang.org/src/crypto/rand/rand.go says ```On Linux, Reader uses getrandom(2) if available, /dev/urandom otherwise.``` so I think we are safe :)